packages/app-admin-ui/client/pages/Home/index.js

```diff
@@ -53,8 +53,12 @@ const HomepageListProvider = () => {
   // TODO: A permission query to limit which lists are visible
   const lists = listKeys.map(key => getListByKey(key));
 
-  const query = gqlCountQueries(lists);
-  const { data, error } = useQuery(query, { fetchPolicy: 'cache-and-network', errorPolicy: 'all' });
+  const query = lists.length !== 0 && gqlCountQueries(lists);
```

Comment:
I think just leave this line as it was. If you really need a conditional, a ternary would be better.

Reply:
The original issue here is that we cannot call gqlCountQueries on an empty array so we have to prevent that from running. I could do this.

```diff
- const query = gqlCountQueries(lists);
+ const query = lists.length !== 0 ? gqlCountQueries(lists) : null;
```

Reply:
I thought the issue was that calling the resulting query in `useQuery` threw the error? So even if you call it on an empty list, since you're using `useLazyQuery`, the invalid query will never be called since the function bails beforehand.

Reply:
The source issue is that Calling gqlCountQueries(lists) on an empty array, witch can be solved by above ternary. However, the following useQuery will then get an invalid query string "null" and to prevent that we can use useLazyQuery.

Reply:
This may have been harder than I thought from the beginning, but to me this looks ok.